lang/java/reef-wake/wake/src/test/java/org/apache/reef/wake/test/time/ClockTest.java

```diff
@@ -118,19 +124,22 @@ public void testMultipleCloseCalls() throws Exception {
     LoggingUtils.setLoggingLevel(Level.FINE);
 
     final int numThreads = 3;
+    final CountDownLatch eventCountLatch = new CountDownLatch(numThreads);
+
     final RuntimeClock clock = buildClock();
     new Thread(clock).start();
     final ThreadPoolStage<Alarm> stage = new ThreadPoolStage<>(new EventHandler<Alarm>() {
       @Override
       public void onNext(final Alarm value) {
         clock.close();
+        eventCountLatch.countDown();
       }
     }, numThreads);
 
     try {
       for (int i = 0; i < numThreads; ++i)
```

Comment:
Good catch! By default Checkstyle ignores test source directories. Once I've enabled Checkstyle for test sources locally, it detected this violation (together with several hundreds of others). I've filed [REEF-487](https://issues.apache.org/jira/browse/REEF-487) for fixing this.